### ISSUE TITLE
Backup and restore containers

### ISSUE THREAD

**Issue by tukusejssirs** [avbhatt/tabs#7]
I would be nice to have containers backed up and restored, although it might not be possible because currently we cannot open a URL in a specific container from terminal/command-line ([mozilla/multi-account-containers #365](https://github.com/mozilla/multi-account-containers/issues/365)). However, I have no idea if it is possible to at least backup them.

**Comment by stephan-dev** [comment]
@tukusejssirs I read that issue's thread (link you provided) and someone made an extension for it after your post : [open-url-in-container](https://addons.mozilla.org/en-US/firefox/addon/open-url-in-container/). It adds a custom protocol to FF.
I'm quite interested in your feature request, too. I don't think I can use "Tab save and restore" extension without containers management, for security and privacy issues.
For some websites I use "always open in (x) container", but not for all.

**Comment by tukusejssirs** [comment]
Do as you think is the best. And thank you for that extension. 👍

BTW, I too use _Always in x container_, but I wish there would be an option to set it for _entire_ URL, not only per host/domain (e.g. I’d like to open a specific Google Sheets document in Work Container, others in the Personal Container). That might be a separate issue. :smiley:

**Comment by avbhatt** [comment]
Thanks for the feedback!

I don't use containers much, so I want to get some clarity on the request: A tab that existed in a specific container should be restored to that same container.

Does that sound right?

**Comment by tukusejssirs** [comment]
@avbhatt, yeah, a tab that is open in a specific container should be open (upon restoration) in the same container (if it exists). But that is just the _process_ how it should work. We might need also a UI where the user could select (all or some) tabs which the user wants to backup. The user should be also able to select which tabs (all/some) should be restored.
